src/guide/build-and-install.md

```diff
@@ -178,6 +179,27 @@ then the issue runs deeper.
 
 :::
 
+## Install OpenSSL
+
+Make sure you have OpenSSL installed. Note that in most Linux setups it is
+already available to you.
+
+- Install OpenSSL on Ubuntu:
+
+  ```bash
+  sudo apt-get install libssl-dev
+  ```
+
+- Install OpenSSL on macOS using [`brew`](https://brew.sh/):
```

Comment:
brew it is displayed not as a link, but as a code.
![image](https://user-images.githubusercontent.com/111361420/186112577-2a59f22f-2ce5-42dc-acf7-ba52ddaa6ff9.png)

Reply:
It's both a link and the inline code
I can make it just a link if you think it's better